docker-compose.yaml

```diff
@@ -372,6 +372,7 @@ services:
             - ${REDIS_PORT}
         volumes:
             - ./local_configs/backend/redis.conf:/usr/local/etc/redis/redis.conf
+        command: ["redis-server", "/usr/local/etc/redis/redis.conf"]
```

Comment:
@slowr could you add this command in redis-fix branch to the proper k8s container place and test? It should have the same effect as in compose. Thanks!